jdaviz/configs/imviz/plugins/parsers.py

```diff
@@ -254,6 +287,35 @@ def _jwst2data(file_obj, ext, data_label):
 
     return data, new_data_label
 
+# ---- Functions that handle input from Roman ASDF files -----
+
+
+def _roman_2d_asdf_to_glue_data(file_obj, data_label, ext=None):
+
+    if ext == '*':
+        ext_list = ['data', 'dq', 'err', 'var_poisson', 'var_rnoise']
```

Comment:
Does this account for all the observing modes or whatever that Roman will churn out?

Reply:
This line was contributed by @tddesjardins, and the background on the available extensions in the `roman_datamodels` docs is rather sparse ([e.g.: this page doesn't seem finished](https://roman-datamodels.readthedocs.io/en/latest/roman_datamodels/datamodels/models.html)). Maybe @tddesjardins can comment?